MSAL/src/MSALPublicClientApplication.m

```diff
@@ -116,16 +131,36 @@ - (id)initWithClientId:(NSString *)clientId
 
 - (NSArray <MSALUser *> *)users:(NSError * __autoreleasing *)error
 {
-    return [_tokenCache getUsers:self.clientId context:nil error:error];
+    NSMutableArray<MSALUser *> *users = [NSMutableArray new];
```

Comment:
I suggest until we agree on changes between users and accounts to keep the previous logic the same, so we can test this step by step. So, this should actually use getAllRTsForClientID and convert it to users.

Reply:
done